orttraining/orttraining/test/python/orttraining_distributed_tests.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import sys
+import argparse
+
+from _test_commons import run_subprocess
+
+import logging
+
+logging.basicConfig(
+    format="%(asctime)s %(name)s [%(levelname)s] - %(message)s",
+    level=logging.DEBUG)
+log = logging.getLogger("Build")
```

Comment:
why is the logger name "Build"? it makes sense for build.py

Reply:
you're right. Let me replace it with a name that makes more sense for these set of tests.